Equipo2Hosteleria/src/main/java/Vista/PanelComandas.java

```diff
@@ -0,0 +1,245 @@
+		textPane_Nombre.setBounds(287, 101, 119, 20);
+		add(textPane_Nombre);
+		
+		textPane_Apellidos = new JTextPane();
+		textPane_Apellidos.setEnabled(false);
+		textPane_Apellidos.setBounds(287, 126, 119, 20);
+		add(textPane_Apellidos);
+		
+		lblTotal = new JLabel("Total:");
+		lblTotal.setBounds(303, 153, 45, 19);
+		add(lblTotal);
+		
+		textPane_Total = new JTextPane();
+		textPane_Total.setBounds(343, 152, 63, 20);
+		add(textPane_Total);
+		
+		scrollPane = new JScrollPane();
+		scrollPane.setBounds(38, 177, 368, 47);
+		add(scrollPane);
+		
+		textPane_Productos = new JTextPane();
+		scrollPane.setViewportView(textPane_Productos);
+		
+		JLabel lblNewLabel = new JLabel("<html><body style=\"text-align:center\"><img src=\"https://i0.wp.com/www.silocreativo.com/wp-content/uploads/2015/03/patron-geometrico-gratis.jpg?resize=600%2C370&quality=100&strip=all&ssl=1\" width=\"700\"/></body></html>");
+		lblNewLabel.setBounds(0, 0, 450, 330);
+		add(lblNewLabel);
+		
+		initializeEvents();
+	}
+	
+	private void initializeEvents() {
+		this.btnVolver.addActionListener(listenerBotonVolver(this.controladorPanelComandas));
+		this.btnDesconectarse.addActionListener(listenerBotonDesconectarse(this.controladorPanelComandas));
+		this.btnPagar.addActionListener(listenerBotonPagar(this.controladorPanelComandas));
+	}
+	
+	public void stateChanged(ChangeEvent e){
+        
+		if (rdbtnTicket.isSelected()) {
+			lblNif.setEnabled(false);
+			lblNombre.setEnabled(false);
+			lblApellidos.setEnabled(false);
+			textPane_NIF.setEnabled(false);
+			textPane_Nombre.setEnabled(false);
+			textPane_Apellidos.setEnabled(false);
+		}
+		if (rdbtnFactura.isSelected()) {
+			lblNif.setEnabled(true);
+			lblNombre.setEnabled(true);
+			lblApellidos.setEnabled(true);
+			textPane_NIF.setEnabled(true);
+			textPane_Nombre.setEnabled(true);
+			textPane_Apellidos.setEnabled(true);
+		}
+	}
+	
+	private ActionListener listenerBotonVolver(ControladorPanelComandas controladorPanelComandas) {
+		return new ActionListener() {
+			public void actionPerformed(ActionEvent arg0) {
+				System.out.println("Ejecutando evento Boton Volver");
+				controladorPanelComandas.accionadoBottonVolverPanelComandas();
+			}
+		};
+	}
+	
+	private ActionListener listenerBotonDesconectarse(ControladorPanelComandas controladorPanelComandas) {
+		return new ActionListener() {
+			public void actionPerformed(ActionEvent arg0) {
+				System.out.println("Ejecutando evento Boton Desconectarse");
+				controladorPanelComandas.accionadoBottonDesconectarsePanelComandas();
+			}
+		};
+	}
+	
+	private ActionListener listenerBotonPagar(ControladorPanelComandas controladorPanelComandas) {
+		return new ActionListener() {
+			public void actionPerformed(ActionEvent arg0) {
+				contentPane = new JPanel();
+				contentPane.setBackground(new Color(0, 153, 255));
+				contentPane.setBorder(new EmptyBorder(5, 5, 5, 5));
+				setContentPane(contentPane);
+				contentPane.setLayout(null);
```

Comment:
La creación de un panel se debe de hacer en una clase aparte como el resto de los paneles. Y, el encargado de realizar la navegación es el controlador. Por lo que, debéis de crear un nuevo controlador y nuevo panel para esta funcionalidad.

Reply:
Arreglado.